src/client/index.js

```diff
@@ -2,43 +2,34 @@
 import React from "react"
 import ReactDOM from "react-dom"
 import Router, { browserHistory } from "react-router"
-import { Provider } from "react-redux"
+import { Provider as ReduxContextProvider } from "react-redux"
 
-import fetchJSON from "../fetchJSON"
-import MetadataProvider from "../MetadataProvider"
+import StatinamicContextProvider from "../ContextProvider"
 
 export default function statinamic({
+  layouts,
   metadata,
   routes,
   store,
 }) {
   let devtools = false
-  if (__DEVTOOLS__) {
+  if (process.env.REDUX_DEVTOOLS) {
```

Comment:
The idea is to avoid this code when builded in production. A test on window. will make this build to have redux devtools code all the time and we don't want that. That's why I used process.env.* which can be replaced by webpack with DefinePlugin. That's said for now I only inject NODE_ENV, so I will add this one and document this.